.github/actions/spelling/expect.txt

```diff
@@ -135,11 +136,11 @@ dsc
 DUPLICATEALIAS
 dustojnikhummer
 dvinns
+ebug
```

Comment:
similarly, this can be avoided by changing: https://github.com/microsoft/winget-cli/blob/d5a5dce3479943db3bdb4f006d80f21ddc8cf344/.github/actions/spelling/patterns.txt#L79

```diff
-(?:^|[\t ,"'`=(])-[D](?!ata|ependency|epth|esktop|estination|ev|irectory|ownload) 
+(?:^|[\t ,"'`=(])-[D](?!ata|ebug|ependency|epth|esktop|estination|ev|irectory|ownload) 
```